docs/docs/configuration.md

```diff
@@ -45,6 +45,8 @@ GRAPH.CONFIG GET OPT1
 GRAPH.CONFIG GET *
 ```
 
+Values set using `GRAPH.CONFIG SET` do not persist after server restart.
```

Comment:
Same comment as above, also consider moving this line to follow:
```
GRAPH.CONFIG SET OPT1 VAL1
```

Reply:
Same